llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -2419,27 +2420,23 @@ X86TargetLowering::X86TargetLowering(const X86TargetMachine &TM,
   }
 
   if (!Subtarget.useSoftFloat() && Subtarget.hasAVX10_2()) {
```

Comment:
We don't need to check VLX here?

Reply:
No, we don't. VLX is a concept of legacy AVX512 features. We have to reuse it for 10.1, but since 10.2, we don't need it anymore.